hobbyloop-api/src/main/java/hobbyloop/backend/api/controller/util/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,23 @@
+package hobbyloop.backend.api.controller.util;
+
+import hobbyloop.backend.domain.EnumTypeBindingException;
+import hobbyloop.backend.domain.ExceptionStatus;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+@Slf4j
+@RestControllerAdvice
+public class GlobalExceptionHandler {
+
+    /**
+     * enum Type 불일치 Exception
+     */
+    @ExceptionHandler(EnumTypeBindingException.class)
+    protected ResponseEntity<ExceptionResponse> handle(EnumTypeBindingException error) {
+        log.error("handle error", error);
```

Comment:
이렇게 하면 `error`가 출력 안되지 않나요? 
```suggestion
        log.error("handle error : {}", error);
```

Reply:
수정 후에
```
2023-05-31 13:00:21.418 ERROR 36539 --- [nio-8080-exec-2] h.b.a.c.util.GlobalExceptionHandler      : handle error : {}

hobbyloop.backend.domain.EnumTypeBindingException: 요청에 대한 Enum Type 을 찾을 수 없음
	at java.base/java.util.Optional.orElseThrow(Optional.java:403)
	at hobbyloop.backend.domain.ticket.TicketSortType.of(TicketSortType.java:23)
	at hobbyloop.backend.domain.ticket.TicketService.navigate(TicketService.java:23)
	at hobbyloop.backend.domain.ticket.TicketService.getTicketsWithRanking(TicketService.java:18)
	at hobbyloop.backend.domain.ticket.TicketService$$FastClassBySpringCGLIB$$5fae869f.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:218)
```
형태로 출력이 돼서 보니 로깅에 들어가는 파라미터의 타입 문제로 보여서
error.getMessage()  를 파라미터로 전달하여 다음과 같이 출력되는 것 확인했습니다
```
2023-05-31 12:59:26.728 ERROR 36487 --- [nio-8080-exec-1] h.b.a.c.util.GlobalExceptionHandler      : handle error : 요청에 대한 Enum Type 을 찾을 수 없음
```